Networking/Networking/Remote/SiteAPIRemote.swift

```diff
@@ -13,7 +13,7 @@ public class SiteAPIRemote: Remote {
     ///   - siteID: Site for which we'll fetch the API settings.
     ///   - completion: Closure to be executed upon completion.
     ///
-    public func loadAPIInformation(for siteID: Int64, completion: @escaping (SiteAPI?, Error?) -> Void) {
+    public func loadAPIInformation(for siteID: Int64, completion: @escaping (Swift.Result<SiteAPI, Error>) -> Void) {
```

Comment:
The `import Alamofire` above is not actually necessary. If we remove that, then we can remove the `Swift.` prefix here. 

```suggestion
    public func loadAPIInformation(for siteID: Int64, completion: @escaping (Result<SiteAPI, Error>) -> Void) {
```

Reply:
Right! Fixed in 2a2f3fa.
I'll make a note to self to revisit other remotes to remove unnecessary imports.